doc/user_guide/statements/merge.md

```diff
@@ -0,0 +1,52 @@
+# MERGE
+
+You can construct [`MERGE`](https://docs.exasol.com/sql/merge.htm) SQL statements using the `Merge` class.
+
+`Merge` supports a combination of `INSERT`, `UPDATE` and `DELETE` where source data is merged into a destination table. The merge strategy is configurable and depends on whether or not a row in source and destination are considered a match.
+Of course the criteria for that match is configurable too.
+
+## Creating `MERGE` Commands
+
+You can create a minimalistic `MERGE` like this:
+
+```java
+final Merge merge = StatementFactory.getInstance()
+    .merge("destination")
+    .using("source")
+    .on(eq(column("source", "id"), column("destination", "id");
+```
+
+### Fine Tuning the `MERGE` Strategies
+
+If you need more control over what happens in case of matching rows, you can add `whenMatched()`.
+
+In case of `whenMatched()` select one of the two strategies `thenInsert()` &hellip;
+
+```java
+merge.whenMatched()
+    .thenUpdate() //
```

Comment:
I'm confused. It's said insert above, but update in the example

Reply:
Fixed.